pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10960,10 +10960,15 @@
   hoc_activities:
     heading: "Explore activities"
     desc: "Teachers: [Host an hour](%{host_url}) or [read the How-To Guide](%{how_to_url})"
-    previous:
-      heading: "Looking for previous years' activities?"
-      desc: "We've moved all previous activities [here](%{archive_url})."
-      button: "View archived activities"
+    archive:
+      heading: "Can't find your favorite activity?"
+      button: "Explore our archives"
```

Comment:
Should this be "archive" to match the nomenclature elsewhere?

Reply:
Can you clarify what part you're referencing?

Reply:
The page URL contains "archive" but the button text says "archives".  It seems like the button should match the URL and be singular rather than plural.

Reply:
Oh I'm not sure, the button copy came from marketing. @dmcavoy should I update this? I haven't shared these strings with Jorge yet so there's time to update it.

Reply:
I think its fine to leave as is

Reply:
I'd like to push back a little.  We put so much effort into `/learn` over the years and attention to details like this was always very important.